core/ui/src/main/java/com/merxury/blocker/core/ui/component/ComponentListItem.kt

```diff
@@ -263,13 +299,42 @@ fun ComponentItemPreview() {
     BlockerTheme {
         Surface {
             ComponentListItem(
-                simpleName = "ExampleActivity",
-                name = "com.merxury.blocker.feature.appdetail.component.ExampleActivity",
-                packageName = "com.merxury.blocker",
+                item = ComponentItem(
+                    simpleName = "ExampleActivity",
+                    name = "com.merxury.blocker.feature.appdetail.component.ExampleActivity",
+                    packageName = "com.merxury.blocker",
+                    description = "An example activity",
+                    type = ComponentType.ACTIVITY,
+                    pmBlocked = false,
+                ),
+                enabled = false,
+                type = SERVICE,
```

Comment:
I think you need to use ACTIVITY as the type. I know it doesn't matter but it is better to do so.